### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by bradurani**

### CODE REVIEW

**Comment by inem** [comment]
Hey, I wasn't expecting it is that bad! Thanks a lot!

File: source/principles.haml_ [hunk 1]
```diff
@@ -59,42 +59,42 @@ layout: page
...
         %p
           If you ever try to do something like this, you quickly start to get all kinds of unpredictable errors and misbehavior.
-          It is just because those gnomes under the hood go crazy of the complexity they need to handle. Or in other words, nobody from core team supposed that it can be used in such an interesting way. And nobody cared to implement that specific case.
-        %p And at this point it becomes your personal problem, not Rails core team problem. And you have two options:
+          It is just because those gnomes under the hood go crazy with the complexity they need to handle. Or in other words, nobody from the core team predicted that it could be used in such an interesting way. And nobody cared to implement that specific case.
+        %p And at this point it becomes your personal problem, not a Rails core team problem. You have two options:
         %ul
           %li
-            jump into ActiveRecord sources, and try to be a hero, who covers that specific scenario and becomes
+            jump into ActiveRecord the source and try to be a hero who covers that specific scenario and becomes a
```
Comment (inem) [inline source/principles.haml_:94]: However this one looks incorrect. I meant source code here

File: source/mess.md.erb [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
 
 This ability can come from two sources:
 
-- Complexity can be managed by the developer if he is familiar with corresponding practices.
-- Or framework itself offers some solutions &amp; instruments to for the same purpose.
+- Complexity can be managed by the developer if she is familiar with best practices.
+- The framework itself offers solutions and tools the suit your purpose.
 
-Rails framework is known to be optimized for developer's convenience. Things can become hairy pretty quick. Remember conditional validations for example.
+the Rails framework is optimized for developer convenience. As your application grows, things become hairy pretty quickly. For example, remember conditional validations?
```
Comment (develop7) [inline source/mess.md.erb:33]: Shouldn't "The" start with capital "T"?

Comment (inem) [inline source/mess.md.erb:33]: fixed, thanks!

**Pull request merged**